Views/Timeline/index.html.php

```diff
@@ -66,22 +88,22 @@
     <?php $view['slots']->output('_content'); ?>
 </div>
 <script>
-    mQuery(function() {
+    mQuery(function () {
```

Comment:
If possible, let's move this JS to the Assets folder. Possibly timeline.js or something?